src/data/javascript-quiz.ts

```diff
@@ -1104,6 +1104,17 @@ const javascriptQuiz = [
       "typeof is a JavaScript keyword that will return the type of a variable when you call it. You can use this to validate function parameters or check if variables are defined.",
     Link: "https://www.freecodecamp.org/news/javascript-data-types-typeof-explained/",
   },
+  {
+    Question:
+      "In Javscript, What is the return value when pushing an element to an array, using .push() method?",
```

Comment:
Please use correct capitalization for JavaScript. Also, let use this rephrasing of the question.

```suggestion
      "In JavaScript, what is the return value of the .push() method?",
```